srcpkgs/dialog/template

```diff
@@ -3,18 +3,28 @@ pkgname=dialog
 _distver=1.3
 _date=20210117
 version="${_distver}.${_date}"
-revision=1
+revision=2
 wrksrc="${pkgname}-${_distver}-${_date}"
 build_style=gnu-configure
-configure_args="--with-ncursesw --disable-nls"
-makedepends="ncurses-devel"
+configure_args="LIBTOOL=${XBPS_CROSS_BASE}/usr/bin/libtool --with-ncursesw --with-libtool --disable-nls"
+make_build_args="LIBTOOL=${XBPS_CROSS_BASE}/usr/bin/libtool"
+make_install_args="LIBTOOL=${XBPS_CROSS_BASE}/usr/bin/libtool"
```

Comment:
Ugh, do we really need to involve libtool so much? :/

Reply:
AFAIK it fails to cross-build without that